fuel-vm/src/tests/crypto.rs

```diff
@@ -601,7 +616,7 @@ fn ed25519_verify_c_gt_vmaxram_sub_32() {
         op::xor(reg_b, reg_b, reg_b),
         op::not(reg_a, RegId::ZERO),
         op::subi(reg_a, reg_a, 31),
-        op::ed19(reg_b, reg_b, reg_a),
+        op::ed19(reg_b, reg_b, reg_a, 32),
```

Comment:
Can we add 2 more tests, one to check 64-byte len, and one where we leave the len blank/empty?

Reply:
Added in f964db61